packages/angular_devkit/build_angular/src/angular-cli-files/utilities/bundle-calculator.ts

```diff
@@ -289,7 +290,7 @@ class AnyCalculator extends Calculator {
       .filter(asset => !asset.name.endsWith('.map'))
       .map(asset => ({
         size: asset.size,
-        label: asset.name,
+        label: relative(process.cwd(), asset.name),
```

Comment:
While I like the idea of having relative paths, I think this needs to be taken a more holistic and generic approach. Currently most if not error errors/warnings are absolute.

Reply:
I can remove that and it can be solved upstream. It did feel a bit hacky since there is no base path available.

Reply:
I think we can do this in the CLI but at a "global" level.

Let's remove this change for the time being.